server/controllers/apply/applicationsController.test.ts

```diff
@@ -140,13 +148,30 @@ describe('applicationsController', () => {
     const submittedApplication = applicationFactory.build({
       person: personFactory.build({ name: 'Roger Smith' }),
       submittedAt: '2024-02-05',
+      timelineEvents: [
+        timelineEventsFactory.build({
+          type: 'cas2_prison_transfer',
+          label: 'Prison transfer',
+          body: 'the status description',
+          createdByName: 'A Nacro',
+          occurredAt: '2025-03-15T15:07:42Z',
+        }),
+      ],
     })
 
     it('renders the overview page', async () => {
       ;(fetchErrorsAndUserInput as jest.Mock).mockImplementation(() => {
         return { errors: {}, errorSummary: [], userInput: {} }
       })
 
+      const timelineEvents = [
+        {
+          label: { text: 'Prison transfer' },
+          byline: { text: 'A Nacro' },
+          datetime: { timestamp: '2025-03-15T15:07:42Z', type: 'datetime' },
+        },
+      ] as UiTimelineEvent[]
+
       applicationService.findApplication.mockResolvedValue(submittedApplication)
```

Comment:
Could you mock `getApplicationTimelineEvents` with the timeline events? We arent to concerned about what shape the timeline events are only that we call service and we forward the response to the view